recipes/pygridtools/meta.yaml

```diff
@@ -0,0 +1,43 @@
+package:
+    name: pygridtools
+    version: 0.2
+
+source:
+    git_url: https://github.com/Geosyntec/pygridtools.git
+    git_tag: v0.2
```

Comment:
Sure, I guess my thought is some git repos can become very huge and then this download step will take time (and some space) versus simply downloading a tarball of a release. After looking more closely, this repo does not seem to be one of those case. So, I leave it up to you.